samples/apps/forum/src/types/ccf.ts

```diff
@@ -74,6 +99,7 @@ export interface CCF {
   ): ArrayBuffer;
 
   kv: KVMaps;
+  state?: State;
```

Comment:
Only for historical queries. Bad name?

Reply:
`historical.state` or `historicalState` would be preferable.